src/api/spec/controllers/webui/obs_factory/staging_projects_controller_spec.rb

```diff
@@ -87,10 +93,13 @@
 
           it { expect(response).to have_http_status(:success) }
           it { expect(response).to render_template(:index) }
-          it { expect(assigns(:backlog_requests_ignored)).to be_empty }
-          it { expect(assigns(:backlog_requests)).to be_empty }
-          it { expect(assigns(:requests_state_new)).to be_empty }
-          it { expect(assigns(:project)).to eq(factory) }
+
+          it 'sets up the required variables' do
+            expect(assigns(:backlog_requests_ignored)).to be_empty
+            expect(assigns(:backlog_requests)).to be_empty
+            expect(assigns(:requests_state_new)).to be_empty
+            expect(assigns(:project)).to eq(factory)
+          end
         end
       end
     end
```

Comment:
> This also reduces the number of VCR cassettes needed.

Why aren't they removed in the commit then?

Reply:
I actually did the commits afterwards, but moved then before the render_view changes after I realized that I was removing cassettes that just got added one or two commits before.